app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -121,32 +123,30 @@ class AppVersionActivityTest {
   @get:Rule
   val oppiaTestRule = OppiaTestRule()
 
-  @get:Rule
-  val activityTestRule: ActivityTestRule<AppVersionActivity> = ActivityTestRule(
-    AppVersionActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
-  )
-
   @Inject
   lateinit var context: Context
 
   @Inject
   lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
 
+  @Inject
+  lateinit var appLanguageLocaleHandler: AppLanguageLocaleHandler
+
   @Before
   fun setUp() {
     Intents.init()
     setUpTestApplicationComponent()
     testCoroutineDispatchers.registerIdlingResource()
   }
 
-  @Test
-  fun testAppVersionActivity_hasCorrectActivityLabel() {
-    activityTestRule.launchActivity(createAppVersionActivityIntent())
-    val title = activityTestRule.activity.title
+  @After
+  fun tearDown() {
+    testCoroutineDispatchers.unregisterIdlingResource()
+    Intents.release()
+  }
 
-    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
-    // correct string when it's read out.
-    assertThat(title).isEqualTo(context.getString(R.string.app_version_activity_title))
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
```

Comment:
Function `setUpTestApplicationComponent` already exists, and is called in `setup()` above. This is a duplicate.